lib/matplotlib/backend_bases.py

```diff
@@ -450,14 +450,14 @@ def draw_image(self, gc, x, y, im, transform=None):
         Parameters
         ----------
         gc : `GraphicsContextBase`
-            a graphics context with clipping information.
+            A graphics context with clipping information.
 
-        x : scalar
-            the distance in physical units (i.e., dots or pixels) from the left
+        x : float
```

Comment:
These may well need to be ints (because this is called after resampling the image to match pixel boundaries).  This needs to be checked, but in the meantime I would leave scalar here.

Reply:
Reverted.